autogen.sh

```diff
@@ -1,10 +1,23 @@
 #!/bin/sh
-
 set -e
+
+oldpwd=$(pwd)
+topdir=$(dirname "$0")
+cd "$topdir"
+
+CURRENT_VERSION_FILE=./build/current-version
+if [ -d .git ] # don't ruin file that might come from build-rpm.sh
+then
+	. ./build/versioning.sh
+	echo $GIT_VER > $CURRENT_VERSION_FILE
+	echo $GIT_REF >> $CURRENT_VERSION_FILE
+fi
+
```

Comment:
Will be done.
configure file is generated with /bin/sh shell. As such it is limited to /bin/sh syntax. For extended/different scripting language (e.g. /bin/bash) it is possible to use autogen.sh
However in this case the script will be updated to match configure (/bin/sh)